make-webpack-config.js

```diff
@@ -79,6 +79,9 @@ module.exports = function(options) {
             path.resolve(__dirname, 'src/host/'),
             path.resolve(__dirname, 'src/shared/')
         ],
+        exclude: [
+            path.resolve(__dirname, 'src/host/js/umd.js')
```

Comment:
This is to solve a problem where webpack was injecting both AMD and CommonJS utility in the generated script so it was basically rendering the umd loader useless